src/webextension/manage/components/item-details.js

```diff
@@ -14,11 +14,14 @@ import Toolbar from "../../widgets/toolbar";
 
 import styles from "./item-details.css";
 
+import * as telemetry from "../../telemetry";
+
 const PASSWORD_DOT = "\u25cf";
 
 function CopyToClipboardButton({text, ...props}) {
   return (
-    <CopyToClipboard text={text}>
+    <CopyToClipboard text={text}
+      onCopy={() => telemetry.recordEvent("itemCopied", "entryDetails")}>
```

Comment:
Nit: the indentation here is a bit off. I think something like the following would be better:

```js
    <CopyToClipboard text={text} onCopy={() => {
      telemetry.recordEvent("itemCopied", "entryDetails");
    }}>
```

Otherwise, the `onCopy` attribute should be lined up below `text`, but that'll probably push us past 80 columns.

Reply:
@jimporter How does that look now? 

https://github.com/mozilla-lockbox/lockbox-extension/blob/7f11411c44ad9fb25d707ced84186198764995ff/src/webextension/manage/components/item-details.js#L21-L29

I feel like I should indent line 25 (`}}>`) since the `Button` is nested within. but it also feels wrong. Your call. 🙃  

(and if there's a style guide I can go read, please point me to it!)